views/post.py

```diff
@@ -34,8 +38,13 @@ def create(self, request, *args, **kwargs):
         try:
             group = Group.objects.get(pk=group)
             if not has_edit_rights(person, group):
+                logger.warning(
+                    f'Person trying to create Post does'
+                    f' not have admin rights for that group'
+                )
                 raise PermissionDenied
         except Group.DoesNotExist:
+            logger.warning(f'Group does not exist with pk={group}')
```

Comment:
Rephrase for clarity:
```suggestion
            logger.warning(f'Group with primary key \'{group}\' does not exist')
```